what_Is_eth-bloom.txt

```diff
@@ -0,0 +1,19 @@
+# Ethereum Bloom Filters 
+
+Bloom filters are a type of data structure that use cryptographic hashes
+to help data stored within them to be retrieved or stored.  They work like other
+data structures, but in a probabilistic way: it allows for false positive
+matches but not false negative matches.  Bloom filters storage space use is
+low relative to other kinds of data structures.
+
+( For more information on Bloom filters, see Wikipedia: https://en.wikipedia.org/wiki/Bloom_filter )
+
+A trie is an a kind of tree data structure for use with dynamic data
+
+Ethereum bloom filters are bloom filters implemented using a "trie" data
```

Comment:
I don't believe this is entirely accurate.  The bloom filter has nothing to do with the trie data other than the fact that some of the data that the bloom filter holds *happens* to also be stored in the ethereum state.

I am of the opinion that mentioning trie structures is only confusing in this context.

Reply:
that makes a lot more sense.

Reply:
(and will make said change)